Packs/SmartSheetITAdmin/Integrations/Smartsheet_IT_Admin/Smartsheet_IT_Admin.yml

```diff
@@ -0,0 +1,247 @@
+    - contextPath: DisableUser
+      description: Command context path
+      type: Unknown
+    - contextPath: DisableUser.status
+      description: 'User status, set to one of the following values: ACTIVE, DECLINED,
+        or PENDING'
+      type: string
+    - contextPath: DisableUser.brand
+      description: User?s brand name.
+      type: string
+    - contextPath: DisableUser.details
+      description: User?s details including id, email, firtsName, lastName, groupAdmin
+        etc.
+      type: string
+    - contextPath: 'DisableUser.email '
+      description: 'User?s email address. '
+      type: string
+    - contextPath: 'DisableUser.errorCode '
+      description: 'Error code in the case of exception.  Example: 404'
+      type: number
+    - contextPath: 'DisableUser.errorMessage '
+      description: 'Error message in the case of exception. Example: ?User not Found?'
+      type: string
+    - contextPath: 'DisableUser.Users.id '
+      description: 'User?s id '
+      type: string
+    - contextPath: DisableUser.instanceName
+      description: Instance name for the Integration.
+      type: string
+    - contextPath: DisableUser.success
+      description: Success status. Can be True or False
+      type: boolean
+  - arguments:
+    - description: SCIM content in JSON format
+      name: scim
+      required: true
+    - description: An optional custom mapping that takes custom values in the SCIM
+        data into the integration.
+      name: customMapping
+    description: This command creates the user based on the scim and custom map passed
+      in argument.
+    name: enable-user
+    outputs:
+    - contextPath: EnableUser
+      description: Command context path
+      type: Unknown
+    - contextPath: EnableUser.brand
+      description: Name of the Integration
+      type: String
+    - contextPath: EnableUser.instanceName
+      description: Name of the instance used for testing
+      type: String
+    - contextPath: EnableUser.success
+      description: Status of the result. Can be true or false.
+      type: Boolean
+    - contextPath: EnableUser.active
+      description: 'Gives the active status of user. Can be true of false. '
+      type: String
+    - contextPath: EnableUser.id
+      description: Value of id passed as argument
+      type: String
+    - contextPath: EnableUser.email
+      description: Value of email ID passed as argument
+      type: String
+    - contextPath: EnableUser.errorCode
+      description: 'HTTP error response code '
+      type: Number
+    - contextPath: EnableUser.errorMessage
+      description: Reason why the API is failed
+      type: String
+    - contextPath: EnableUser.details
+      description: Gives the user information if the API is success else error information
+      type: String
+  dockerimage: demisto/smartsheet:1.0.0.9945
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+fromversion: 5.0.0
+tests:
+- No tests (auto formatted)
```

Comment:
are we going to have an instance for this integration?

Reply:
We have an instance in the dev environment but it was for free trial. We will create a new one going forward.